### DISCUSSION TITLE
Rotating/oscillating boundary condition for cylinder

### DISCUSSION THREAD

**Discussion by AyushBoral** [xcompact3d/Incompact3d#214] [Q&A] [answered]
Is there any way to implement rotating or oscillating boundary condition at the cylinder wall for the 3D circular cylinder case?

**Comment by slaizet** [comment, marked as answer]
Hi,
Yes it is possible, see for instance https://github.com/xcompact3d/Incompact3d/blob/master/examples/Cylinder/input_DNS300_LR_MOVING.i3d for an example of a moving 3D cylinder. We also tried oscillating cylinders in https://link.springer.com/article/10.1007/s10494-022-00364-4.
See also https://www.sciencedirect.com/science/article/pii/S0307904X21003103 for more details about the approach.
Best,
Sylvain

**Reply by AyushBoral** [reply]
Thank you very much for the support!!
